examples/webhook.ts

```diff
@@ -0,0 +1,176 @@
+import { SpvWalletClient, WebhookManager, WebhookHttpHandler, RawEvent } from '../dist/typescript-npm-package.cjs.js';
+import { exampleAdminKey } from './example-keys.js';
+import { errMessage } from './utils.js';
+// import http from 'http';
+// import express, { Response, Request } from 'express';
```

Comment:
I think we don't have to show both examples.
In my opinion, we should focus on one framework:
- converting to other is rather trivial
- the commented-out code makes it is hard to read this example
- if You prefer, leave the fastify example and remove the express